### PR TITLE
Project Bootstrap

### PR DESCRIPTION
**Pull request by stephenwf**
@giorgiosironi

### CODE REVIEW

**Comment by stephenwf** [comment]
@giorgiosironi Is there a way to get proofreader in the vendor directory so we can run ./vendor/bin/proofreader or something similar?

File: composer.json [hunk 1]
```diff
@@ -2,5 +2,12 @@
     "name": "elife/bus-sdk",
     "description": "eLife Sciences bus SDK",
     "type": "library",
-    "license": "MIT"
+    "license": "MIT",
+    "require": {
```
Comment (giorgiosironi) [inline composer.json]: shouldn't it be `require-dev` so that it isn't inherited by packages that depend on this?

File: tests/bootstrap [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (giorgiosironi) [inline tests/bootstrap]: should have a `.php` extension usually

File: project_tests.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+rm -f build/*.xml
+proofreader src/ tests/
+vendor/bin/phpunit --log-junit build/phpunit.xml
+git status
```
Comment (giorgiosironi) [inline project_tests.sh]: what does the `git status` do?

File: project_tests.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+rm -f build/*.xml
```
Comment (giorgiosironi) [inline project_tests.sh]: needs you to commit with `-f` a `build/.gitkeep` file

File: tests/src/EmptyTest.php [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+class EmptyTest extends \PHPUnit_Framework_TestCase
+{
+  public function testThisIsJustAPlaceholderForRealTestsToCheckTheWholeBuildIsRunningCorrectly()
```
Comment (giorgiosironi) [inline tests/src/EmptyTest.php]: indentation is usually 4 spaces in most of eLife's PHP projects?

File: project_tests.sh (hunk 3 above)
Comment (stephenwf) [inline project_tests.sh]: Typo!

File: project_tests.sh (hunk 4 above)
Comment (stephenwf) [inline project_tests.sh]: Added!

**Comment by stephenwf** [comment]
Cool, fixed those changes, my IDE defaults to Drupal standards so fixed that too 😆

**Comment by giorgiosironi** [comment]
@stephenwf `proofreader` is provided system-wide on the PATH so you should just use `proofreader`

**Pull request merged**